packages/desktop-client/playwright.config.js

```diff
@@ -1,11 +1,39 @@
-module.exports = {
+import { expect, defineConfig } from '@playwright/test';
+
+// Disable screenshot assertions in regular e2e tests;
+// only enable them when doing VRT tests
+if (!process.env.VRT) {
+  expect.extend({
+    toHaveScreenshot() {
+      return {
+        message: () => 'passed',
+        pass: true,
+      };
+    },
+  });
+}
+
+// eslint-disable-next-line import/no-unused-modules
+export default defineConfig({
   timeout: 20000, // 20 seconds
   retries: 1,
   testDir: 'e2e/',
   use: {
+    userAgent: 'playwright',
     screenshot: 'on',
     browserName: 'chromium',
     baseURL: process.env.E2E_START_URL ?? 'http://localhost:3001',
     trace: 'on-first-retry',
+    ignoreHTTPSErrors: true,
   },
-};
+  expect: {
+    toHaveScreenshot: {
+      // Approx. 2% difference (1000 pixels)
+      maxDiffPixels: 1000,
+    },
+    toMatchSnapshot: {
+      // 2% difference is acceptable
+      maxDiffPixelRatio: 0.02,
```

Comment:
I think this is the right place to start. 
However, it does seem possible we could change single element borders, colors, or shadows without failing at 2%. 🤔 

I don't have a better idea for now, though. Just thinking. It would be nice if the snaps perhaps could be higher fidelity, if that would decrease the false positive rate here.

Reply:
Yea, ideally we wouldn't need this.. My recommended strategy would be: start with a 2% difference and then circle back in 2-3 months time when a new docker version is released again. If the new version again creates differences in output - keep this around -, but if the new docker version does not introduce regressions - remove these lines.